ci/Jenkinsfile

```diff
@@ -39,6 +39,7 @@ pipeline {
         stage('relay-reactjs-chat') { steps { script { buildExample() } } }
         stage('store-reactjs-chat') { steps { script { buildExample() } } }
         stage('web-chat') { steps { script { buildExample() } } }
+        stage('noise-js') { steps { script { buildExample() } } }
```

Comment:
This is incorrect and it broke the CI. Noise is an html example so it should be added to line 53 using the script `copyExample`. Cc @weboko

Reply:
Issue was in a wrong folder name
Fixing it here https://github.com/waku-org/js-waku-examples/pull/196